src/tests/functional/test_competitions.py

```diff
@@ -12,6 +13,15 @@ def setUp(self):
         self.user = UserFactory(password='test')
         self.login(self.user.username, 'test')
 
+    def current_server_time_exists(self):
+        # Get server time element
+        element = self.find('#server_time')
+        text = element.get_attribute('innerText')
+
+        # Check that the text is a valid datetime by loading it with strptime.
+        # This will raise a ValueError if the format is incorrect.
+        datetime.strptime(text, '%B %d, %Y, %I:%M %p %Z')
```

Comment:
could you do an assertion of some kind here? at least:

```datetime.strptime(text, '%B %d, %Y, %I:%M %p %Z')```

Reply:
Was this comment left mistakenly unfinished?

Reply:
I left out `assert` in my example :( oops

Reply:
Done. Repassed selenium tests.